docs_src/pipelinesetup_dlstreamer.md

```diff
@@ -1,4 +1,4 @@
-# Set up Pipeline
+# Set up Pipeline (DLStreamer is deprecated)
```

Comment:
We can consider removing all DLStreamer specific documentation.

Reply:
I think it will help reduce confusion. We can always retrieve them from github history if they are needed in the future.